t/110-my_test.t

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env tarantool
```

Comment:
Filename is `t/110-my_test.t`

Please, consider changing it to more appropriate

Reply:
D'oh. Thanks.

Reply:
Fixed